mansplainer.py

```diff
@@ -52,9 +52,12 @@ def incoming_sms():
             model = appC.models.get("general-v1.3")
             imageresult = model.predict_by_url(url=image_URL)
 
-            mansplanation = None
-            for tag in imageresult["outputs"][0]["data"]["concepts"]:
-                mansplanation = wiki.getSummaryFromWiki(tag["name"])
+            resultList = imageresult["outputs"][0]["data"]["concepts"]
+            selectedConcept = random.choice(resultList)
```

Comment:
It would be better in terms of tag relevance, but I thought if it returned the same tag for the same photo, it would be boring. Does Clarifai return probabilities with each tag? That way we can select based on a tag's weighted probability.